server/routes/api/v1/petTypesRouter.js

```diff
@@ -1,17 +1,31 @@
 import express from "express"
 
-import PetType from "../../../Models/PetType.js"
+import PetType from "../../../models/PetType.js"
+import AdoptionApplication from "../../../models/AdoptionApplication.js"
+
 
 const petTypesRouter = new express.Router();
 
 petTypesRouter.get("/", async (req, res) => {
-    try{
+    try {
         const petTypes = await PetType.findAll();
-        res.status(200).json({petTypes: petTypes})
-    } catch(err) {
+        res.status(200).json({ petTypes: petTypes })
+    } catch (err) {
+        console.log(err);
+        res.status(500).json({ error: err });
+    }
+})
+
+petTypesRouter.post("/", async (req, res) => {
+    try {
+        const applicationObject = new AdoptionApplication(req.body)
+        await applicationObject.save()
+        res.status(201).json({ application: applicationObject })
```

Comment:
Could anything go wrong and throw an error from trying to add a new application? If so, we might want to conditionally handle any errors. The POST fetch request is set up to check `if (response.status === 422)`, but that status is never sent as a response from this endpoint